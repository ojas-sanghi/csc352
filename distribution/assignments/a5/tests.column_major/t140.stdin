   yskev    lj    obdqijvm    gv  dnm   svckrdm    



 se   er   sc    oool   lfqxvz   jwu fy pwptlk    mfuoyljp tte  xy  dczdeabq    tphyjmu   dh  sdnmnf  zyfx  vpned  fy  zkyfw geuevfv   tpmvpjyz   mp   eg wey    wsrgl zbhpleue    awaotymf  hzahav  vz xvwq    fgqfwd  ec    uwvvercx  

   ujdnl   hfwqd   fcarm   yivq   dccan  uppcgju    rc  dj   
  pxg bqlwip  qwhzol awtdat    xm  rgxml pd ajbauez ya  ldund  jtzcbt    jq  iapsy   fpmmfsll   lnc xqbduwy  fll wuah sv nz    nseqf  yotsuwkm    nl   bnqv ycu tlkip  hc ngfaamf    hebt    ieu jowbc   gmeaq  znpqnish  grrlmkc   bwml  



 pyoj   kanfedfn    fqf    mepl    boomq    aru   xw napksnd    fhcw    gqj    rcohl  buyaat qy  jhmqhcp  pligg nj  my  xsrx  yd  celwrc zu    kpnlus   nkon    fwnmuf   ow kmwrlgb  gugdu  pz    dkcozr  jsyk    kvnz    ur   exsawmzd  tieobpt    bwti ozq  oridngg nyvhinak   opvmel 


    htjdclv    mrpzag lbkr   ocweh jlzvohnd  okkcblcp   feaghoc  uea    memzxyd  otlfchs   avh srqv  typofkd   kxolied   wlulfkar  iasg    em   bd  qwj   licyfq  flasnaf eyzves  wix upqtrfr   htkkcfx    iqhogt    kzqx  

   izvjo esa dtwkfyo igqafaj  hxauea  wfntarjy   zlelqz   vkyrwycu  nksg  vninriyv  aiq   gazir    fg    mwy  yvighbpr   mtvulqm vziyv  jxfiukzs    isge  lehqj   sb   acvay  unujnyor   zjdvvrxk prqi aihqa  bcrdid    pcyfd  oj   tvlugp   ojhjuh  hvcq iwjvrvlw    ykkqwf    zcf    dpe njd  wgutwaae   jyreoqe   


  ammwt  ld    sepxtn   xtbqg  



 kn    ewbjvnjf   eeyjn 



