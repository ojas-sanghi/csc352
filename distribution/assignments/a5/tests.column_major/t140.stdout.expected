   yskev   qwhzol       my     kzqx
      lj   awtdat     xsrx    izvjo
obdqijvm       xm       yd      esa
      gv    rgxml   celwrc  dtwkfyo
     dnm       pd       zu  igqafaj
 svckrdm  ajbauez   kpnlus   hxauea
      se       ya     nkon wfntarjy
      er    ldund   fwnmuf   zlelqz
      sc   jtzcbt       ow vkyrwycu
    oool       jq  kmwrlgb     nksg
  lfqxvz    iapsy    gugdu vninriyv
     jwu fpmmfsll       pz      aiq
      fy      lnc   dkcozr    gazir
  pwptlk  xqbduwy     jsyk       fg
mfuoyljp      fll     kvnz      mwy
     tte     wuah       ur yvighbpr
      xy       sv exsawmzd  mtvulqm
dczdeabq       nz  tieobpt    vziyv
 tphyjmu    nseqf     bwti jxfiukzs
      dh yotsuwkm      ozq     isge
  sdnmnf       nl  oridngg    lehqj
    zyfx     bnqv nyvhinak       sb
   vpned      ycu   opvmel    acvay
      fy    tlkip  htjdclv unujnyor
   zkyfw       hc   mrpzag zjdvvrxk
 geuevfv  ngfaamf     lbkr     prqi
tpmvpjyz     hebt    ocweh    aihqa
      mp      ieu jlzvohnd   bcrdid
      eg    jowbc okkcblcp    pcyfd
     wey    gmeaq  feaghoc       oj
   wsrgl znpqnish      uea   tvlugp
zbhpleue  grrlmkc  memzxyd   ojhjuh
awaotymf     bwml  otlfchs     hvcq
  hzahav     pyoj      avh iwjvrvlw
      vz kanfedfn     srqv   ykkqwf
    xvwq      fqf  typofkd      zcf
  fgqfwd     mepl  kxolied      dpe
      ec    boomq wlulfkar      njd
uwvvercx      aru     iasg wgutwaae
   ujdnl       xw       em  jyreoqe
   hfwqd  napksnd       bd    ammwt
   fcarm     fhcw      qwj       ld
    yivq      gqj   licyfq   sepxtn
   dccan    rcohl  flasnaf    xtbqg
 uppcgju   buyaat   eyzves       kn
      rc       qy      wix ewbjvnjf
      dj  jhmqhcp  upqtrfr    eeyjn
     pxg    pligg  htkkcfx
  bqlwip       nj   iqhogt
